### PR TITLE
Feature/ndvi web reponsiveness

### LINKED ISSUE #1464: Crash in NVDI component if min property is missing in API response
## Error in Plant-for-the-Planet App

**TypeError** in **/donate-trees**
Cannot read property 'min' of null

[View on Bugsnag](https://app.bugsnag.com/plant-for-the-planet/plant-for-the-planet-app/errors/5d8b1b82a9f0e000199ac192?event_id=5d8b1b8200503a1883490000&i=gh&m=ci)

## Stacktrace

    http://0.0.0.0:8080/bundle.js:147348 - GradientResultLine.calculateHighlightLineColor

[View full stacktrace](https://app.bugsnag.com/plant-for-the-planet/plant-for-the-planet-app/errors/5d8b1b82a9f0e000199ac192?event_id=5d8b1b8200503a1883490000&i=gh&m=ci)

*Created automatically via Bugsnag*

### PR DESCRIPTION
**Pull request by hkurra**
https://drive.google.com/file/d/1NJqRTP97bfRXjI-oCApJatX95uFbx9PH/view?usp=sharing

also fixes #1464

### CODE REVIEW

**Comment by hkurra** [comment]
https://drive.google.com/file/d/1NJqRTP97bfRXjI-oCApJatX95uFbx9PH/view?usp=sharing

**Comment by norbertschuler** [comment]
You can see the error at the bottom of the output log in CircleCI:
```
ERROR in ./web/widgets/NDVI/ndvi.widget.scss
Module build failed: 
  @include for-phone-only {
          ^
      No mixin named for-phone-only
      in /home/circleci/ttc-app/web/stylesheet/NDVI/ndvi-container.scss (line 19, column 12)
 @ ./web/widgets/NDVI/widget.js 15:0-29
 @ multi babel-polyfill ./web/widgets/NDVI/widget.js

ERROR in ./web/widgets/NDVI/ndvi.widget.scss
Module build failed: ModuleBuildError: Module build failed: 
  @include for-phone-only {
          ^
      No mixin named for-phone-only
      in /home/circleci/ttc-app/web/stylesheet/NDVI/ndvi-container.scss (line 19, column 12)
    at runLoaders (/home/circleci/ttc-app/node_modules/webpack/lib/NormalModule.js:195:19)
    at /home/circleci/ttc-app/node_modules/loader-runner/lib/LoaderRunner.js:367:11
    at /home/circleci/ttc-app/node_modules/loader-runner/lib/LoaderRunner.js:233:18
    at context.callback (/home/circleci/ttc-app/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at Object.asyncSassJobQueue.push [as callback] (/home/circleci/ttc-app/node_modules/sass-loader/lib/loader.js:55:13)
    at Object.done [as callback] (/home/circleci/ttc-app/node_modules/neo-async/async.js:8067:18)
    at options.error (/home/circleci/ttc-app/node_modules/node-sass/lib/index.js:294:32)
 @ ./web/widgets/NDVI/ndvi.widget.scss
 @ ./web/widgets/NDVI/widget.js
 @ multi babel-polyfill ./web/widgets/NDVI/widget.js
```

**Comment by norbertschuler** [comment]
@hkurra can you please add a description of what this pull request is about so I know what to test in a review?

**Comment by hkurra** [comment]
> @hkurra can you please add a description of what this pull request is about so I know what to test in a review?

https://drive.google.com/file/d/1NJqRTP97bfRXjI-oCApJatX95uFbx9PH/view?usp=sharing

File: web/config/webpack.dev.config.js [hunk 1]
```diff
@@ -21,10 +21,6 @@ module.exports = webpackMerge(commonConfig, {
     progressbarwidget: [
       'babel-polyfill',
       path.join(__dirname, '../widgets/progressbar/widget.js')
-    ],
-    ndviwidget: [
```
Comment (norbertschuler) [inline web/config/webpack.dev.config.js]: Why do you remove this plugin. Wasn't is part of the NDVI feature?

**Comment by norbertschuler** [comment]
Hints are partially hidden either on small or large screens and one of the (?) buttons as well - see my screen shots:

Bildschirmfoto 2019-09-25 um 10 00 37
Bildschirmfoto 2019-09-25 um 10 02 07
Bildschirmfoto 2019-09-25 um 10 02 34
Bildschirmfoto 2019-09-25 um 10 03 04

**Comment by norbertschuler** [comment]
I optimized the hints so they are at least completely shown on larger screens. I have not solved the problem for smaller screens as the default behavior of the `ReactTooltip` component does not support it by default. 
I have made the Captured CO2 be shown as a nice card on mobile as well.

File: web/config/webpack.dev.config.js (hunk 1 above)
Comment (norbertschuler) [reply web/config/webpack.dev.config.js]: My decision for now would be to leave out the widget, but keep the code.

**Pull request merged**